.github/workflows/UnitTests.yml

```diff
@@ -94,14 +108,6 @@ jobs:
       run: |
         docker run -v /home/runner/actions-runner/_work/maxtext/maxtext:/app --rm --privileged maxtext_base_image bash -c \
         'python3 MaxText/decode.py MaxText/configs/base.yml run_name=runner_$(date +%Y-%m-%d-%H-%M)-${RANDOM} base_output_directory=gs://runner-maxtext-logs dataset_path=gs://maxtext-dataset steps=2 ici_tensor_parallelism=4 attention=dot_product enable_checkpointing=false max_target_length=128 per_device_batch_size=.25'
-    - name: Test standalone_dataloader.py
```

Comment:
I'm happy for you to make this change but think about what you're doing. We want standalone_dataloader to work on TPUs. Right now you're protected. With this change you wouldn't be. Seems like a tradeoff IMO.

Reply:
Thanks Rafi, I believe that standalone_dataloader and standalone_checkpointer tests would be the same on TPUs and CPUs. I will keep them on both for some time, to see if I observe any differences.